examples/mutators/apply-setters/simple/README.md

```diff
@@ -55,17 +55,17 @@ kind: MyKind
 metadata:
   name: ns
 environments: # kpt-set: ${env}
-- prod
-- stage
+  - prod
+  - stage
 ```
 
 ## Function invocation
 
 Get the config example and try it out by running the following commands:
 
 ```sh
-kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/mutators/apply-setters/simple .
-kpt fn run simple
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/mutators/apply-setters/simple
+$ kpt fn eval simple/ --image gcr.io/kpt-fn/apply-setters:unstable --fn-config simple/fn-config.yaml
```

Comment:
@mengqiy This falls under https://github.com/GoogleContainerTools/kpt/issues/1548

What's the general approach here, I expected this migration to be done holistically, not piecemeal.

Reply:
Can we unblock this PR.

Reply:
We should do the migration all at once when we approaching CLI v1.0 which is weeks away from now.

@phanimarupaka Can we keep using `kpt fn run`? Are there any issues?

Reply:
Done.